Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -69,16 +76,24 @@ public class BTShopperInsightsClient {
                 isVenmoRecommended: venmo?.recommended ?? false,
                 isEligibleInPayPalNetwork: payPal?.eligibleInPayPalNetwork ?? false || venmo?.eligibleInPayPalNetwork ?? false
             )
-            return self.notifySuccess(with: result)
+            return self.notifySuccess(with: result, for: experiment)
         } catch {
-            throw self.notifyFailure(with: error)
+            throw self.notifyFailure(with: error, for: experiment)
         }
     }
 
     /// Call this method when the PayPal button has been successfully displayed to the buyer.
     /// This method sends analytics to help improve the Shopper Insights feature experience.
-    public func sendPayPalPresentedEvent() {
-        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.payPalPresented)
+    /// - Parameters:
+    ///    - paymentMethodsDisplayed: Optional:  The list of available payment methods, rendered in the same order in which they are displayed i.e. ['Apple Pay', 'PayPal']
+    ///    - experiment: Optional:  A `JSONObject` passed in as a string containing details of the merchant experiment.
```

Comment:
Do you have more detail on what this is for? Why would a merchant need to pass in raw JSON?

Reply:
yep this was an ask from product - they were wanting to track payment ready conversion across both test and control groups (it seems like DD may be running multiple tests in parallel and so they suggested adding this extra param)